vcr/credential/resolver.go

```diff
@@ -39,7 +39,7 @@ func FindValidatorAndBuilder(credential vc.VerifiableCredential) (Validator, Bui
 			}
 		}
 	}
-	return defaultCredentialValidator{}, nil
+	return defaultCredentialValidator{}, defaultBuilder{vcType: vc.VerifiableCredentialType}
```

Comment:
Fix the function description:
```// When no additional type is provided, it returns the default validator and a nil builder```

Reply:
Looks like the defaultBuilder is completely unused except for tests. All the return values of the `Builder` type of the FindValidatorAndBuilder method are ignored as well.